bat/BLDJAR.BAT

```diff
@@ -32,7 +35,8 @@ copy  src\z390.man jar
 cd jar
 rem compile z390 java sources to class vm object code
 java -version
-javac.exe -g:none *.java
+if /I %1. EQU .Debug. javac.exe -g:none *.java
+if /I %1. NEQ .Debug. javac.exe -g      *.java
```

Comment:
EQU and NEQ should be exchanged. See https://docs.oracle.com/en/java/javase/18/docs/specs/man/javac.html.